CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 in development
 --------------
 
+Changed:
```

Comment:
If we are following some kind of standard them I'm kinda OK with it even though "Changes" is a bit weird once since it's more like a parent category since every other thing (fix, security) is also a change that's why I would prefer "Improvements" and "New features" and a categories instead.